book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,121 +1,389 @@
+
+
+
+
+
+# How to make hardware open source
+
+
+In order to make your hardware reusable and modifiable by others, its source should be shared with an appropriate license.
+This is usually done via specific online platforms (see {ref}`rr-open-hardware-platforms`.
+The type and amount of shared content depends on the complexity of the OSH, and on the importance of the community aspects of the project.
+
+In the following sections, you can find descriptions of the common types of content that you should consider sharing, such as project documentation, technical documentation, and community interactions.
+You are not required to post or document them all, but the more you share, the more the community benefits, and the more it can give back.
+
+
+There is a lot of crossover with files to include in open source software projects 
+(see {ref}`rr-rdm-metadata-documentation`), and the community aspects that are common to any open source project are described elsewhere  (see {ref}`pd-overview-repro`)
+
+
+
+
+
+## Project documentation
+
+Your OSH project should at a minimum provide a license {ref}`rr-licensing-hardware` together with a README file with all the basic, general information a newcomer needs to get oriented.
+Include a general description of the hardware’s identity and purpose, written as much as possible for a general audience.
+That is, explain what the project is and what it is for, before you get into the technical details.
+
+
+On top of what is mentioned in detail in the open source project documentation page ({ref}`pd-overview-repro`) for project plan, people involved, and contribution process, 
+you should specifically **think about your audience** when writing OSH documentation.
+Indeed,
+your project might be reused by people with different skills, roles, objectives, and socio-economic and cultural environments.
+Because of this it can be useful to create a **list of skills** that are required to build or use the hardware.
+Someone trying to build it from scratch, for example, will require specific set of prior knowledge, skills, and tools.
+A different set is needed to perform maintenance tasks.
+An end user operating the assembled project might require an entirely distinct skillset (and documentation).
+
+**Take particular care about safety instructions. OSH makers are not always formally trained engineers and may not be able to easily differentiate between dangerous and safe manipulations.**
+
+Your project documentation may also include a functional overview of the project's parts or modules, as well as a short description of the software needed to use the hardware.
+Also give an overview of the state of the hardware, software and documentation (current state, ongoing development, and/or any future plans for the project), and other information you think may be useful for newcomers.
+
+
+<!--  
```

Comment:
Lines 185 to 204 are outdated commentary and should be removed

Reply:
would be good to keep the info somewhere, maybe in a issue (here or by us ?)

Reply:
Either way is fine by me, it just shouldn't be in the main document. Thanks!